src/utils.c

```
@@ -191,7 +221,7 @@ int parse_cuda_visible_env() {
     for (int i = 0; i < CUDA_DEVICE_MAX_COUNT; i++) {
         LOG_INFO("device %d -> %d",i,cuda_to_nvml_map(i));
     }
-    LOG_INFO("get default cuda from %s", getenv("CUDA_VISIBLE_DEVICES"));
+    LOG_INFO("get default cuda from %s", vgpu_getenv("CUDA_VISIBLE_DEVICES"));
```

Cppcheck (2.21.0)

[warning] If resource allocation fails, then there is a possible null pointer dereference

(nullPointerOutOfResources)